server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceConfig.java

```diff
@@ -285,6 +291,7 @@ private VS originalValuesSource() {
             if (valueSourceType() == ValuesSourceType.BYTES) {
                 return (VS) bytesScript();
             }
+            // TODO: Do we need a range script case?
```

Comment:
Probably, eventually.  I suspect it's non-trivial work though, so OK marking that as a limitation for first pass and we can revisit.